.github/actions/upload_test_artifacts/action.yaml

```diff
@@ -0,0 +1,19 @@
+name: Test Artifact Upload
+description: Uploads test archives to GCS for on-device tests.
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Cloud SDK
+      uses: isarkis/setup-gcloud@40dce7857b354839efac498d3632050f568090b6 # v1.1.1
+    - name: Upload Artifacts to GCS
+      env:
+        WORKFLOW: ${{ github.workflow }}
+      run: |
+        set -eux
+        project_name=$(gcloud config get-value project)
+        for x in `find "${GITHUB_WORKSPACE}"/src/out/${{ matrix.platform }}_${{ matrix.config }}/apks -name "*.apk"`
```

Comment:
The test apks are not in the `apks` folder but rather in folders next to it. You could also make this simpler by using globbing:

```
gsutil cp "${GITHUB_WORKSPACE}"/src/out/${{ matrix.platform }}_${{ matrix.config }}/**/*.apk" \
  gs://"${project_name}"-test-artifacts/"${WORKFLOW}"/"${GITHUB_RUN_NUMBER}"/${{matrix.platform}}/
```

Reply:
done